package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "AdminLTE",
+  "version": "1.0.0",
+  "private": true,
+  "description": "",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pi-hole/AdminLTE.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/pi-hole/AdminLTE/issues"
+  },
+  "homepage": "https://github.com/pi-hole/AdminLTE#readme",
+  "scripts": {
+    "xo": "xo",
```

Comment:
One can automatically autofix the issues with `npm run xo -- --fix`. If you prefer to have a separate script for this let me know how you want it named and I'll add it.

Reply:
Should this be something we encourage contributors to do, or something that CI just does and pushes back to the PR?

Reply:
This is useful for people fiddling with the source code. CI will just run the normal script